feat: adds configure amtpassword local command

Craig-Spencer-12 opened the pull request:
https://github.com/open-amt-cloud-toolkit/rpc-go/pull/441

`./rpc configure amtpassword`
`./rpc configure amtpassword -password [current password]`
`./rpc configure amtpassword -newamtpassword [new password]`
`./rpc configure amtpassword -password [current password] -newamtpassword [new password]`

Info that is not provided in a flag should be prompted for

matt-primrose reviewed (changes requested):
Hi Craig, please address the following items:

- Need to add the new command and examples of use to the configure help in the printConfigurationUsage function
- I'd like to have the filename amtPass.go and amtPass_test.go spelled out completely amtPassword.go and amtPassword_test.go.  
- When I enter the wrong password for the existing AMT password, I get a dump of wsman (when not using -v or -l parameter).  Instead, we should have an error message that states that the AMT Password provided was incorrect.

Craig-Spencer-12 commented:
> Hi Craig, please address the following items:
> 
> * Need to add the new command and examples of use to the configure help in the printConfigurationUsage function
> * I'd like to have the filename amtPass.go and amtPass_test.go spelled out completely amtPassword.go and amtPassword_test.go.
> * When I enter the wrong password for the existing AMT password, I get a dump of wsman (when not using -v or -l parameter).  Instead, we should have an error message that states that the AMT Password provided was incorrect.

1. Fixed the printConfigurationUsage function
2. Renamed the files
3. This functionality is consistent with main and across all configure commands. I can look into how to change it if you think this is an issue, but that would be a separate PR
